File: 3_2/3_2.js
function countEvenDigits(number) {
    let count = 0;
    while (number > 0) {
        let digit = number % 10;
        if (digit % 2 === 0) {
            count++;
        }
        number = Math.floor(number / 10);
    }
    return count;
}

function main() {
    let number;
    let numbers = [];
    console.log("Enter the numbers (enter 0 to complete):");
    while (true) {
        number = parseInt(prompt());
        if (number === 0) {
            break;
        }
        numbers.push(number);
    }
    for (let num of numbers) {
        let numEvenDigits = countEvenDigits(num);
        console.log("Number of even digits in " + num + ": " + numEvenDigits);
    }
}

main();

File: 3_2/3_2.rb
def count_even_digits(number)
    count = 0
    while number > 0
        digit = number % 10
        if digit % 2 == 0
            count += 1
        end
        number = number / 10
    end
    return count
end

numbers = []
puts "Enter the numbers (enter 0 to complete):"
while true
    number = gets.chomp.to_i
    if number == 0
        break
    end
    numbers.push(number)
end

numbers.each do |num|
    num_even_digits = count_even_digits(num)
    puts "Number of even digits in #{num}: #{num_even_digits}"
end